recipe_sha: 0766f18bb08493be1a82072d25fda9929831be9b6bd6b626c80679858d94f4e1  # The commit at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-
   - r-dt
   - bioconductor-affxparser

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package ascat-3.0.0-r41hdfd78af_0 requires r-base >=4.1,<4.2.0a0, but none of the providers can be installed
  -

  Leaving build/test directories:
    Work:
   /opt/conda/conda-bld/work
    Test:
   /opt/conda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 244, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 234, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - bioconductor-biostrings
   - bioconductor-aroma.light
   - r-sequenza
   - r-foreach
   - bioconductor-copynumber
   - r-doparallel
   - r-changepoint
   - bioconductor-bsgenome.hsapiens.1000genomes.hs37d5
   - bioconductor-rhdf5
   - bioconductor-genomicranges
   - r-dplyr
   - r-data.table
   - r-rmarkdown
   - r-mclust
   - bioconductor-limma
   - ascat >=3
   - r-iotools
   - r-facets
   - r-bedr
   - r-base 4.3.*
   - bioconductor-rsamtools
   - r-seqinr
   - r-dt
   - bioconductor-affxparser

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package ascat-3.0.0-r41hdfd78af_0 requires r-base >=4.1,<4.2.0a0, but none of the providers can be installed
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2038, in build
      output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 787, in expand_outputs
      for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 2524, in get_output_metadata_set
      conda_packages = finalize_outputs_pass(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 884, in finalize_outputs_pass
      fm = finalize_metadata(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 547, in finalize_metadata
      build_unsat, host_unsat = add_upstream_pins(m,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 409, in add_upstream_pins
      host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(m, 'host',
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 374, in _read_upstream_pin_files
      deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 131, in get_env_dependencies
      actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("ascat==3.0.0=r41hdfd78af_0"), MatchSpec("r-base[version='>=4.1,<4.2.0a0']")}
# Last 100 lines of the build log.
